libraries/standard/json/src/json.c

```diff
@@ -217,18 +217,17 @@ static bool_ skipUTF8( const char * buf,
 {
     bool_ ret = false;
 
-    if( *start < max )
+    assert( *start < max );
```

Comment:
Moved the check to an assert @dan4thewin

Reply:
Why was this change made?

Reply:
because it is not falsifiable with any input so we cannot achieve 100% branch coverage without this change

Reply:
Oh ok. Got it. Thanks